jcloud/api.py

```diff
@@ -204,18 +205,18 @@ async def remove(args):
 
         flow_id_list = args.flows
 
-    # Case 3: remove all ALIVE flows.
+    # Case 3: remove all ALIVE and FAILED flows.
     else:
         if 'JCLOUD_NO_INTERACTIVE' not in os.environ:
             confirm_deleting_all = Confirm.ask(
-                f'[red]Are you sure you want to delete ALL the ALIVE flows that belong to you?[/red]',
+                f'[red]Are you sure you want to delete ALL the ALIVE and FAILED flows that belong to you?[/red]',
                 default=True,
             )
             if not confirm_deleting_all:
                 print('[cyan]No worries. Exiting...[/cyan]')
                 return
 
-        _raw_list = await _list_by_phase(Phase.Serving.value, name='')
+        _raw_list = await _list_by_phase(phase=None, name='')
```

Comment:
Better send the phases `Serving,Failed` here, instead of assuming default values in `_list_by_phase` if it is None.